kpops/cli/pipeline_config.py

```diff
@@ -1,11 +1,18 @@
+from __future__ import annotations
+
 from pathlib import Path
+from typing import TYPE_CHECKING, Any
 
 from pydantic import BaseConfig, BaseSettings, Field
-from pydantic.env_settings import SettingsSourceCallable
 
 from kpops.component_handlers.helm_wrapper.model import HelmConfig, HelmDiffConfig
 from kpops.utils.yaml_loading import load_yaml_file
 
+if TYPE_CHECKING:
+    from collections.abc import Callable
```

Comment:
shouldn't this be
```suggestion
    from typing import Callable
```
?

Reply:
importing it from `typing` is deprecated in favor of `collections.abc` I believe. [link](https://docs.python.org/3.10/library/typing.html#callable)

Reply:
TIL, good point